multiversx_sdk_cli/validators/validators_file.py

```diff
@@ -34,6 +35,19 @@ def load_signers(self) -> List[ValidatorSigner]:
 
         return signers
 
+    def load_public_keys(self) -> List[ValidatorPublicKey]:
+        public_keys: List[ValidatorPublicKey] = []
+
+        for validator in self.get_validators_list():
+            # Get path of "pemFile", make it absolute
+            validator_pem = Path(validator.get("pemFile")).expanduser()
+            validator_pem = validator_pem if validator_pem.is_absolute() else self.validators_file_path.parent / validator_pem
+
+            pem_file = ValidatorPEM.from_file(validator_pem)
```

Comment:
These 3 lines can also be extracted to a separate function e.g. `load_validator_pem(path): ValidatorPEM` / optional (code exists above, as well).

Reply:
extracted in a method